src/ru/nsu/ccfit/zuev/osu/menu/SongMenu.java

```diff
@@ -741,7 +741,7 @@ public void setFilter(final String filter, final SortOrder order,
                           final boolean favsOnly, Set<String> limit) {
         String oldBeatmapPath = "";
         if (selectedBeatmap != null) {
-            oldBeatmapPath = selectedBeatmap.getPath();
+            oldBeatmapPath = selectedBeatmap.getFilename();
```

Comment:
Since this represents beatmap filename now, can it be renamed to `oldBeatmapFilename` along with the methods that use it?

Reply:
Solved with [965432a](https://github.com/osudroid/osu-droid/pull/394/commits/965432ad545916e37fb64e9e5b0148f816b01368)